exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserServer.java

```diff
@@ -197,7 +197,8 @@ protected MessageLite getResponseDefaultInstance(int rpcType) throws RpcExceptio
   }
 
   /**
-   * {@link AbstractRemoteConnection} implementation for user connection. Also implements {@link UserClientConnection}.
+   * {@link org.apache.drill.exec.rpc.AbstractRemoteConnection} implementation for user connection.
```

Comment:
Nit: The super class/interface is clear from the generated Javadoc; that need not be explained here. What is helpful is that which cannot be extracted from the code: what is the purpose of this class? What the heck is a "Bit to user connection"? Is this where the Drillbit connects to a neural implant in the user's brain? The Drillbit connects back to the user's client? Something else?